### DISCUSSION TITLE
local book on top? (android)

### DISCUSSION THREAD

**Discussion by skyzuma** [advplyr/audiobookshelf#1918] [General]
maybe is only my behavior but on my android phones downloaded books are on the bottom of all ... i think the best option is to pull them on the top?

1. continue listening
2. local  books
3. recently added
4. ...

**Comment by advplyr** [comment]
There is a feature request for this in the audiobookshelf-app repo already

**Reply by skyzuma** [reply]
sry for the wrong section, i realised it to late
